### DISCUSSION TITLE
AdminLTE 3.1.0

### DISCUSSION THREAD

**Discussion by TheBauerShell** [ColorlibHQ/AdminLTE#3467] [Q&A]
i first case thanks for your incredible work.

in your 3.1.0 RC version you have got an Excamplepage Kanban Board.
what is your Idea to get it to work in the background. is there any js module you would suggest to use?

kind regards Totto Bauer (TheBauerShell)

**Comment by REJack** [comment]
At the moment thats a simple html/css example. For „sorting“ the cards I would use Dragula or Sortable.js. A fully working example will come with AdminLTE v4.

**Reply by TheBauerShell** [reply]
thanks :)
